README.md

```diff
@@ -458,23 +458,41 @@ The streaming API will receive deltas in real time, without needing to repeatedl
 ```ruby
 cursor = nylas.latest_cursor
 
-last_cursor = nil
-nylas.delta_stream(cursor) do |event, object|
-  if event == "create" or event == "modify"
-    if object.is_a?(Nylas::Contact)
-      puts "#{object.name} - #{object.email}"
-    elsif object.is_a?(Nylas::Event)
-      puts "Event!"
+EventMachine.run do
+  nylas.delta_stream(cursor) do |event, object|
+    if event == "create" or event == "modify"
+      if object.is_a?(Nylas::Contact)
+        puts "#{object.name} - #{object.email}"
+      elsif object.is_a?(Nylas::Event)
+        puts "Event!"
+      end
+    elsif event == "delete"
+      # In the case of a deletion, the API only returns the ID of the object.
+      # In this case, the Ruby SDK returns a dummy object with only the id field
+      # set.
+      puts "Deleting from collection #{object.class.name}, id: #{object}"
     end
-  elsif event == "delete"
-    # In the case of a deletion, the API only returns the ID of the object.
-    # In this case, the Ruby SDK returns a dummy object with only the id field
-    # set.
-    puts "Deleting from collection #{object.class.name}, id: #{object}"
   end
-  last_cursor = object.cursor
+end
+```
+
+In order to receive streams from multiple accounts, simply create multiple API handles (one for each account), then open a `delta_stream` for each of them.
```

Comment:
This isn't very clear. Maybe heva seomthing like "To receive streams from multiple accounts, call `delta_stream` for each of them inside an `EventMachine.run` block.